doc/Command-Reference.md

```diff
@@ -1732,7 +1732,7 @@ This command displays information regarding port-channel interfaces
   
 **show interface status**  
 
-This command displays some more fields such as Lanes, Speed, MTU, Type, Asymmetric PFC status and also the operational and administrative status of the interfaces 
+This command displays some more fields such as Lanes, Speed, MTU, Type, Asymmetric PFC status, Neighbor device , Neighbor port and also the operational and administrative status of the interfaces 

```

Comment:
Remove space between "device" and the comma.

Reply:
removed additional space